src/TodoList.jsx

```diff
@@ -1,11 +1,13 @@
+import TodoListItem from "./TodoListItem";
+
 function TodoList(){
     const todos = [
         {id: 1, title: "review resources"},
         {id: 2, title: "take notes"},
         {id: 3, title: "code out app"},
     ]
-
-    return <ul>{todos.map(todo => <li key={todo.id}>{todo.title}</li>)}</ul>
+//<li key={todo.id}>{todo.title}</li>
```

Comment:
Don't forget to remove comments when you are finished with them. It helps keep the codebase clean and easy to skim.